database-plugins/src/main/java/co/cask/hydrator/plugin/DBConfig.java

```diff
@@ -62,4 +62,9 @@
   public DBConfig() {
     jdbcPluginType = "jdbc";
   }
+
+  protected String cleanQuery(String query) {
+    query = query.trim();
+    return query.endsWith(";") ? query.substring(0, query.length() - 1) : query;
```

Comment:
do you think we should handle multiple semi-colons? Or is that overkill?

Also, is the problem with only semicolons or with a set of characters at the end?